Feature/commands wizard commit (#209)
HPInc/ml-git: docs/mlgit_commands.md

```
Usage: ml-git datasets add [OPTIONS] ML_ENTITY_NAME [FILE_PATH]...

>   Add datasets change set ML_ENTITY_NAME to the local ml-git staging area. [This command has a wizard that will request the necessary information if they are not informed]

Reviewer: ```suggestion
  Add datasets change set ML_ENTITY_NAME to the local ml-git staging area. [This command has a wizard that will request the necessary information if it is not informed]
```

Author: The suggestion was applied in https://github.com/HPInc/ml-git/commit/e5a21ae38fa1458efdf49bd36dffbefea6b83e10.

Usage: ml-git models commit [OPTIONS] ML_ENTITY_NAME

  Commit model change set of ML_ENTITY_NAME locally to this ml-git

>   repository. [This command has a wizard that will request the necessary information if they are not informed]

Reviewer: ```suggestion
  repository. [This command has a wizard that will request the necessary information if it is not informed]
```

Author: The suggestion was applied in https://github.com/HPInc/ml-git/pull/209/commits/e5a21ae38fa1458efdf49bd36dffbefea6b83e10.

```
Usage: ml-git repository remote datasets add [OPTIONS] REMOTE_URL

>   Add remote dataset metadata REMOTE_URL to this ml-git repository. [This command has a wizard that will request the necessary information if they are not informed]

Reviewer 2: Please check if this command has been modified, I believe it has not.

>   Add remote dataset metadata REMOTE_URL to this ml-git repository. [This command has a wizard that will request the necessary information if they are not informed]

Author: The suggestion was applied in https://github.com/HPInc/ml-git/pull/209/commits/d65158d225f88c873671cfa0f57288bf006a1e93
